lms/templates/ux/reference/pattern-library-test.html

```diff
@@ -0,0 +1,30 @@
+## Override the default styles_version to the Pattern Library version (version 2)
+<%! main_css = "css/lms-main-v2.css" %>
+
+<%page expression_filter="h"/>
+<%inherit file="/main.html" />
+
+<%block name="pagetitle">Pattern Library Test</%block>
+<%block name="nav_skip">#content</%block>
+
+<%block name="bodyclass">pattern-library</%block>
+
+<%block name="content">
+<h3 class="hd-6 example-set-hd">Warning Alert with Message Only</h3>
+<div class="example-set">
+    <div class="alert alert-warning" role="alert" aria-labelledby="alert-warning-publish-title" tabindex="-1">
+        <span class="icon alert-icon icon-warning" aria-hidden="true"></span>
+
+        <div class="alert-message">
+            <h3 class="sr-only alert-title" id="alert-warning-publish-title">
+                Interesting pattern library content to come...
+            </h3>
+
+            <p class="alert-copy">
+                Interesting pattern library content to come...
+            </p>
```

Comment:
Nitpick because it's only a test page but the alert title should be different from the copy, otherwise a screenreader user will hear the same thing twice. In this case I don't think we beed the title.